src/rust/core/server/src/threads/admin.rs

```diff
@@ -329,6 +353,147 @@ impl Admin {
         };
     }
 
+    fn human_stats(&self) -> String {
+        let mut data = Vec::new();
+
+        for metric in &metrics::common::metrics::metrics() {
+            let any = match metric.as_any() {
+                Some(any) => any,
+                None => {
+                    continue;
+                }
+            };
+
+            if let Some(counter) = any.downcast_ref::<Counter>() {
+                data.push(format!("{}: {}", metric.name(), counter.value()));
+            } else if let Some(gauge) = any.downcast_ref::<Gauge>() {
+                data.push(format!("{}: {}", metric.name(), gauge.value()));
+            } else if let Some(heatmap) = any.downcast_ref::<Heatmap>() {
+                for (label, value) in PERCENTILES {
+                    let percentile = heatmap.percentile(*value).unwrap_or(0);
+                    data.push(format!("{}_{}: {}", metric.name(), label, percentile));
+                }
+            }
+        }
+
+        data.sort();
+        data.join("\n")
+    }
+
+    fn json_stats(&self) -> String {
```

Comment:
Add a short description/template to show what json stats format means, since it is conforming to a particular Twitter convention (shall we add a link to the finagle metrics format too?)